test/evm_script.js

```diff
@@ -279,5 +279,26 @@ contract('EVM Script', accounts => {
                 })
             })
         })
+
+        context('script overflow', async () => {
+            const encodeCallScriptBad = actions => {
+                return actions.reduce((script, { to, calldata }) => {
+                    const addr = rawEncode(['address'], [to]).toString('hex')
+                    const length = rawEncode(['uint256'], [calldata.length]).toString('hex') // should be (calldata.length - 2) / 2
```

Comment:
Could we expand this comment, to more fully explain why this causes a bad encoding?